bindgen/codegen/mod.rs

```diff
@@ -3791,7 +3797,8 @@ impl TryToRustTy for Type {
                 // sizeof(NonZero<_>) optimization with opaque blobs (because
                 // they aren't NonZero), so don't *ever* use an or_opaque
                 // variant here.
-                let ty = fs.try_to_rust_ty(ctx, &())?;
+                let ty = fs
+                    .try_to_rust_ty(ctx, &self.name().map(|x| x.to_string()))?;
```

Comment:
I don't think you need to pass a name here. The name wouldn't be what you want anyways because this is generating a field, not an actual function.

So I think you can simplify back `try_to_rust_ty` and so on, and just pass `None` in `FunctionSig::try_to_rust_ty`. Test still passes locally with that as expected.